Bookish/Bookish.DataAccess/LibraryService.cs

```diff
@@ -23,9 +23,10 @@ public LibraryService(IDbConnection connection)
             databaseConnection = connection;
         }
 
-        public IEnumerable<Book> GetBooks()
+        public IEnumerable<Book> GetBooks(string search)
         {
-            return databaseConnection.Query<Book>("SELECT * FROM Books");
+            var sql = "SELECT * FROM Books WHERE Books.title LIKE @searchString OR Books.authors LIKE @searchString";
+            return databaseConnection.Query<Book>(sql, new{searchString = "%" + search + "%"});
```

Comment:
The should be whitespace after new, after the opening curly bracket and before the closing curly bracket. And you can use string interpolation for the searchString:
```
searchString = $"%{search}%"
```
🙂